backend/sites/src/app/middleware/userJwtTokenDecoder.ts

```diff
@@ -22,6 +22,7 @@ export class UserJWTTokenDecoderMiddleware implements NestMiddleware {
         const { email, sub, identity_provider, given_name, family_name } =
           decodedToken;
 
+        req['user'] = decodedToken;
```

Comment:
As discussed we already get user information using "@AuthicatedUser()" we don't need to set it again in request. Please remove it.